### ISSUE TITLE
Midpoint Review VA Notify Forms Strike Team, In Progress Reminders]

### ISSUE THREAD

**Issue by megansiddle** [department-of-veterans-affairs/va.gov-team#39454]
# Steps to complete Midpoint Review 

## Before meeting

**VFS Product Manager**
- [x]  Create this issue and schedule meeting via Calendly (you are responsible for adding the VFS attendees). 
- [x] Notify the Collaboration Cycle team of this ticket with scheduled date/time in #vfs-platform-support ([see Slack Workflow User guide](https://depo-platform-documentation.scrollhelp.site/support/Getting-help-from-the-Platform-in-Slack.1439138197.html)). 
- [x]  Add artifacts listed below to the ticket at least 2 days before scheduled meeting. 

**Platform**
- [ ] Add the Platform reviewers to scheduled meeting
- [ ] Add Zoom information to scheduled meeting (if synchronous)
- [ ] Create feedback tickets and link to the Midpoint Review epic (if no feedback, please comment stating that). 

## After meeting
**VFS Product Manager**
- [x] Respond to feedback tickets.
- [x] Close this ticket when review is complete and feedback has been addressed.  

**Platform**
- [ ] Update [Platform Collaboration Point Tracker](https://docs.google.com/spreadsheets/d/1OgPyEvUlNF6EnaYMFAXJkV6FKOvZnlPnbOQ2fAJ7W7A/edit#gid=266151061). 

### Has your team met with the Sitewide Content team for content support? 
- [x] Yes* We haven't met yet but submitted a ticket for their review
- [ ] No

\*If yes, please include any relevant content artifacts.

## Artifacts

Provide links or documents for the following: 

- [x] [Product outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/va.gov-strike-team/in-progress-reminders.md)
- [x] [Finalized HTML email design](https://images.zenhubusercontent.com/5bbd3157d467817a2f85fcf4/b1cd4b08-7a04-43b9-b1b5-6589f17687bd) [Finalized Plain Text design](https://user-images.githubusercontent.com/10037396/161135892-98d2a604-d88a-4c17-bc30-e0ce0e51323b.png) 
- [ ] Research plan 

 
- [ ] Conversation guide 

### Content artifacts 

- [ ] Your product’s content source of truth
- [ ] Github ticket for any relevant static content page and entry point updates for tool or feature (if applicable)
- [Content & IA review ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/39389)

### QA artifacts

[Learn more about QA artifacts](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/quality-assurance/qa-artifacts.md#test-plan)

- [ ] Regression test plans (if there isn't already a regression test plan for this feature in TestRaiProduct)
- [x] Link to test cases/test plan in [TestRail](https://dsvavsp.testrail.io/index.php?/plans/view/3120)

## Asynchronous review option
Would you like this review to be asynchronous? 

Please refer to the [Midpoint Review guidance](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Midpoint-review.1781039167.html) for the difference between a synchronous meeting and an asynchronous review.
- [x] Yes
- [ ] No

## Scheduling
- You will need to use Calendly to schedule both synchronous and asynchronous reviews (asynchronous reviews will not have a Zoom link).
- Open the [Calendly midpoint review calendar](https://calendly.com/collaboration-cycle/design-intent-or-midpoint-review)
- Select a date and time and click “Confirm”
- Add your name and email
- Click "Add Guests" and enter the VFS meeting attendees email addresses. 
- Click Schedule Event

## Additional information
Please refer to [Platform Collaboration Cycle](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/index.html) or the [Midpoint Review Touchpoint](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Midpoint-review.1781039167.html) on Platform website for more information about the Collaboration Cycle.

FYI: @shiragoodman , @chaseakirby

**Comment by chaseakirby** [comment]
The async review has been scheduled for 4/5, meaning you will receive feedback by EOD. No meeting will be held.

**Comment by briandeconinck** [comment]
Just noting that I will not have my accessibility feedback submitted by end of day today, but it should be in later tonight. (Child care situation has my work schedule thrown off this week.)

cc @shiragoodman @chaseakirby

**Comment by claytonzook** [comment]
@megansiddle checking in about research artifacts, or expected timeline for research, before I leave research feedback.

**Comment by megansiddle** [comment]
Hi @claytonzook we have opted to leverage some existing research around notifications and there frequencies. Some of the key findings are included below. 

A few findings from the Opt Out Research:
Participants would like to manage their cadence of notifications, the types of notifications they get, the medium of communication all in one location.

“I would like to manage all my communication preferences whether it's physical hard copy mail, email, etc. in one spot.”
“I would like to manage the cadence - daily update, weekly update - all notifications from all sources, to review, frequency...sometimes people spam you out with too many notifications.”
“Rather than calling or finding it online it'll make it so much easier to do it this way [notification center].”

5 out of 12 participants mentioned that notifications can be overkill.

Participants mentioned appointment reminders can come through a lot of different mediums and on different cadences but they can also appreciate how hard it is to miss an appointment.
“They'll send them two weeks out, one week out, and the day prior which is a lot. It'd be hard to miss an appointment.”
“It was frustrating and overwhelming with all the information.”
One participant mentioned that in particular at the beginning of enrollment she felt spammed out by the number of notifications from the VA she would get all at once, and felt she missed out on benefits as a result.
“I would have rather done it in a newsletter type format in one email versus getting 10-15 emails one after the other on a wednesday at 10am during a workweek, on a monthly basis. It was so overwhelming that I just deleted them and missed out on benefits because of the way information was just shoved down my throat."
